deployments/liqo/README.md

```diff
@@ -64,7 +64,7 @@
 | gateway.imageName | string | `"ghcr.io/liqotech/liqonet"` | Image repository for the network gateway pod. |
 | gateway.metrics.enabled | bool | `false` | Expose metrics about network traffic towards cluster peers. |
 | gateway.metrics.port | int | `5872` | Port used to expose metrics. |
-| gateway.metrics.serviceMonitor.enabled | bool | `false` | Enable/Disable a Prometheus servicemonitor. |
+| gateway.metrics.serviceMonitor.enabled | bool | `false` | Enable/Disable a Prometheus servicemonitor. Turn on this flag when the Prometheus Operator  runs in your cluster; otherwise simply export the port above as an external endpoint. |
```

Comment:
```suggestion
| gateway.metrics.serviceMonitor.enabled | bool | `false` | Enable/Disable a Prometheus servicemonitor. Turn on this flag when the Prometheus Operator runs in your cluster; otherwise simply export the port above as an external endpoint. |
```